### ISSUE TITLE
Accessible name is required in the Nav examples: ariaLabel should be provided if nav is not the only one used on a page

### ISSUE THREAD

**Issue by jlukosky** [OfficeDev/office-ui-fabric-react#10331]
### Environment Information

- **Package version(s)**: (fill this out)
- **Browser and OS versions**: (fill this out)

> Note that our screen reader support is limited to Edge + Narrator. Issues that do not reproduce in Edge + Narrator, and aren't caused by obvious invalid aria values, should be filed with the respective screen reading software, not the Fabric repo.

### Describe the issue:

1. In the provided examples of the Nav control, ariaLabel should be used since there are multiple nav landmarks on the page. This helps screen readers know the context of the navigation. I realize these are only demonstrations but I think it should be added to these so the best use is shown. In addition, if a page only has one nav landmark, it is still fine to give it an accessible name.
2. The Best Practices should be updated with guidance on providing the accessible name (through the ariaLabel). You can find out more about the requirements by reading [Navigation Landmark: ARIA Landmark Example](https://www.w3.org/TR/2017/NOTE-wai-aria-practices-1.1-20171214/examples/landmarks/navigation.html)

### Please provide a reproduction of the issue in a codepen:
Nope, not in codepen. This relates to improving the current examples at the Fabric [Nav control page](https://developer.microsoft.com/en-us/fabric#/controls/web/nav).

#### Actual behavior:

Only first two instances of the `nav` landmark use an accessible name that is needed by assistive technology. None of the examples given use an accessible name as required and guidance for the accessible name is omitted from the Best Practices.

List of all landmarks and their accessible name where provided

|Landmark type|Accessible name (if provided)| Accessible name required |
|----------------|------------------------------|--------------------------------|
|  **Navigation**     |     Contextual menu         | Yes |
|   **Navigation**   |     All microsoft menu      | Yes |
|  **Navigation**   |                                            | Yes (missing) |
|  Main            |                                            | No |
|  **Navigation**   |                                            | Yes (missing) |
|  **Navigation**   |                                            | Yes (missing) |
|  **Navigation**   |                                            | Yes (missing) |
|  **Navigation**   |                                            | Yes (missing) |
| Content Information |                                | No|
| **Navigation**    |  Footer Resource links        | Yes (missing) |
|  **Navigation**   | Microsoft corporate links   | Yes (missing) |

#### Expected behavior:
1. Since multiple `nav` landmarks are used, each should have an accessible name.
2. Since this site's purpose is to assist users learn good methods of implementing Fabric controls it would be a good idea to include accessible name guidance in the Best Practices.

### Documentation describing expected behavior

- [Navigation Landmark: ARIA Landmark Example](https://www.w3.org/TR/2017/NOTE-wai-aria-practices-1.1-20171214/examples/landmarks/navigation.html)
- [Implementing ARIA Landmarks for the Web (Microsoft Corpnet)](https://microsoft.sharepoint.com/teams/msenable/Pages/Implementing-ARIA-landmarks-for-the-web.aspx#NavigationRole)

**Comment by maxwellred** [comment]
Hi @jlukosky thank you for such a thorough a11y issue! I'll start looking at Nav shortly.

**Comment by maxwellred** [comment]
Closing per the PR merge above #10358
